style.css

```diff
@@ -0,0 +1,72 @@
+:root{
+    --main-background-color: #ffdd99;
+    --other-background-color: #336600;
+    --text-color: #ffdd99;
+    --nice-black-color: #001a13;
+}
+
+.align-center {
+    text-align: center;
+}
+
+html{
```

Comment:
please everywhere space before the {, because when you will be using linter, you will get error/warning for this

Reply:
so, you say i should use this:
.cls
{
...
}

Reply:
No. Space after "html" in this case.
html { } instead of html{}